packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -2216,11 +2216,14 @@ abstract class BuildContext {
   /// be called apply to this method as well.
   InheritedWidget dependOnInheritedElement(InheritedElement ancestor, { Object? aspect });
 
-  /// Obtains the nearest widget of the given type `T`, which must be the type of a
-  /// concrete [InheritedWidget] subclass, and registers this build context with
-  /// that widget such that when that widget changes (or a new widget of that
-  /// type is introduced, or the widget goes away), this build context is
-  /// rebuilt so that it can obtain new values from that widget.
+  /// Returns the nearest widget of the given type `T` and creates a dependency
+  /// on it, or null if no appropriate widget is found.
+  ///
+  /// The widget found will be a concrete [InheritedWidget] subclass, and
+  /// calling [dependOnInheritedWidgetOfExactType] registers this build context
+  /// with the returned widget such that when that widget changes (or a new
+  /// widget of that type is introduced, or the widget goes away), this build
+  /// context is rebuilt so that it can obtain new values from that widget.
```

Comment:
nit: this is sort of a giant run-away sentence. Maybe break it up a little?

Reply:
Done.